core/src/main/scala/kafka/common/InconsistentControllerIdException.scala

```diff
@@ -0,0 +1,10 @@
+package kafka.common
+
+/**
+ * Indicates the controllerId stored in logDirs is not consistent across logDirs.
+ */
+class InconsistentControllerIdException(message: String, cause: Throwable) extends RuntimeException(message, cause) {
+  def this(message: String) = this(message, null)
+  def this(cause: Throwable) = this(null, cause)
+  def this() = this(null, null)
+}
```

Comment:
nit: missing trailing newline

Reply:
Reverted to using InconsistentBrokerMetadataException